grommet/grommet
# Add defaultProps export for typescript usage

#### What does this PR do?
This adds an export statement in the index.d.ts file to allow defaultProps and extendDefaultTheme to be used in Typescript projects.

#### Where should the reviewer start?
src/js/index.d.ts

#### What testing has been done on this PR?
Testing has been done in a local typescript project by trying to import `{ defaultProps } from 'grommet'` prior to adding this import statement as well as after. Prior, there was an error that there was no exported module in grommet with this name, but afterwards it works and the defaultProps can be used.

#### How should this be manually tested?
In a local typescript project, attempt to import `{ defaultProps } from 'grommet'` with the existing code and expect to see a flag in your editor. Then, use this change and expect to see that the flag has been removed.

#### Any background context you want to provide?
In Typescript projects, the defaultProps couldn't be used in an import statement.

#### What are the relevant issues?
#3470 

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
No.

#### Should this PR be mentioned in the release notes?
No.

#### Is this change backwards compatible or is it a breaking change?
Yes, this change is backwards compatible.
